bioblend/galaxy/tool_dependencies/__init__.py

```diff
@@ -0,0 +1,82 @@
+"""
+Contains interactions dealing with Galaxy dependency resolvers.
+"""
+from bioblend.galaxy.client import Client
+
+
+class ToolDependenciesClient(Client):
+
+    def __init__(self, galaxy_instance):
+        self.module = 'dependency_resolvers'
+        super().__init__(galaxy_instance)
+
+    def summarize_toolbox(self, index=None, tool_ids=None, resolver_type=None, include_containers=False, container_type=None, index_by='requirements'):
+        """
+        Summarize requirements across toolbox (for Tool Management grid).
+
+        :type index: int
+        :param index: index of the dependency resolver
```

Comment:
```suggestion
        :param index: index of the dependency resolver, with respect to the
            ``container_resolvers_conf.xml`` config file.
```

Reply:
I think this probably only works for dependency resolvers so I'll make this 'with respect to the dependency resolvers config file', will update now